Cargo.toml

```diff
@@ -20,9 +20,9 @@ lazy_static = { version = "1.3.0", features = ["spin_no_std"] }
 rand = { version = "0.8.0", features = ["std_rng"], default-features = false }
 byteorder = { version = "1.3.1", default-features = false }
 subtle = { version = "2.1.1", default-features = false }
-simple_asn1 = { version = "0.5", optional = true }
-pem = { version = "0.8", optional = true }
 digest = { version = "0.9.0", default-features = false }
+pkcs1 = { version = "0.2.2", default-features = false }
+pkcs8 = { version = "0.7.4", default-features = false }
 
 [dependencies.zeroize]
 version = "=1.3" # zeroize 1.4 is MSRV 1.51+
```

Comment:
if we bump MSRV to 1.51, lets update this as well

Reply:
Bumped MSRV in 2f2a67388137129357ab518953a8cc7fad5fc490.

I went with a less restrictive `>=1, <1.5` requirement to ensure `rsa` can be used alongside crates which are locking `=1.3`. Otherwise it wouldn't be possible to use both /cc @newpavlov